course-mw/course-actors-common/src/main/java/org/sunbird/learner/actors/course/dao/impl/ContentHierarchyDaoImpl.java

```diff
@@ -0,0 +1,42 @@
+package org.sunbird.learner.actors.course.dao.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.Constants;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.RequestContext;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.actors.course.dao.ContentHierarchyDao;
+import org.sunbird.learner.util.Util;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ContentHierarchyDaoImpl implements ContentHierarchyDao {
+
+    private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+    private Util.DbInfo contentHierarchyDb = Util.dbInfoMap.get(JsonKey.CONTENT_HIERARCHY_STORE_DB);
+    private ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public List<Map<String, Object>> getProgramChildrens(RequestContext requestContext, String programId) {
```

Comment:
let's rename this method to -> getContentChildren to make it more generic

Reply:
Done